criu/cgroup.c

```diff
@@ -1947,6 +1947,20 @@ static int prepare_cgroup_sfd(CgroupEntry *ce)
 	return 0;
 }
 
```

Comment:
static int cgroupd_unblock_sigterm(void)
{

Reply:
Thanks for pointing this out, fixed.